templates/kontena-agent.env.j2

```diff
@@ -0,0 +1,73 @@
+#
+# {{ ansible_managed }}
+#
+
+# Set URL of your kontena server
+KONTENA_URI={{ ka_master_uri|default('ws://127.0.0.1:8080') }}
+
+# Set Kontena token
+KONTENA_TOKEN={{ ka_token|default('secret_token') }}
+
+# Kontena peer interface (overlay network interface)
+KONTENA_PEER_INTERFACE={{ ka_peer_interface|default('secret_token') }}
```

Comment:
As I see, ka_peer_interface is default variable, so we could omit  "default" here. Or, much better, remove this variable from defaults, as it's value is not such necessary for kontena, but set it default value "eth1" could potentially cause some troubles.

Reply:
Okay, I'm fixed typo and omit "default" out of here. But default value of variable in official documentation equal to `eth1`.